notifier/notifier.go

```diff
@@ -126,14 +103,45 @@ func New(s *storage.Storage, concurrency int, logger *log.Logger, dwnlURL string
 
 // Start starts the Notifier loop and instruments the worker goroutines that
 // perform the actual notify requests.
-func (n *Notifier) Start(closeChan chan struct{}) {
+func (n *Notifier) Start(closeChan chan struct{}, cfg map[string]map[string]interface{}) {
+	ctx, cancelfunc := context.WithCancel(context.Background())
+
+	for id, v := range cfg {
+		if len(v) == 0 {
+			continue
+		}
+
+		if id == HTTP {
+			n.backends[id] = &httpbackend.Backend{}
+		} else if id == KAFKA {
+			n.backends[id] = &kafkabackend.Backend{}
+		}
+
+		b := n.backends[id]
+		n.Log.Printf("\nStarting %s backend", b.ID())
```

Comment:
Why is the starting newline needed? There should already be one before this print statement.